canonical-websites/vanillaframework.io
# Update the Vanilla dependency to v.1.6.3

## Done
Update the Vanilla dependency to v.1.6.3 with a hotfix for a max-width bug.

## QA
- Pull code
- Run `./run`
- Go to http://0.0.0.0:8014/
- Click around and check everything is looking good

## Issue / Card
https://github.com/vanilla-framework/vanilla-framework/issues/1471
